specification/ai/Azure.AI.Unified/main.tsp

```diff
@@ -0,0 +1,70 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./agents/routes.tsp";
+import "./agents/threads/routes.tsp";
+import "./agents/messages/routes.tsp";
+import "./agents/runs/routes.tsp";
+import "./agents/run_steps/routes.tsp";
+import "./agents/files/routes.tsp";
+import "./agents/streaming/events.tsp";
+import "./agents/vector_stores/routes.tsp";
+import "./agents/vector_stores/files/routes.tsp";
+import "./agents/vector_stores/file_batches/routes.tsp";
+import "./connections/routes.tsp";
+import "./evaluations/routes.tsp";
+import "./datasets/routes.tsp";
+import "./models/routes.tsp";
+import "./indexes/routes.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+namespace Azure.AI {
+
+}
+
+@useAuth(
+  OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit,
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/v2.0/authorize",
+      scopes: ["https://management.azure.com/.default"],
+    }
+  ]>
+)
+@service({
+  title: "Azure AI Data Plane",
```

Comment:
Just "Azure AI" here please, as per our existing TypeSpec... Other things get appended at the end in package metadata, and this works best with just "Azure AI".
https://github.com/Azure/azure-rest-api-specs/pull/32705/files#diff-b721009c98a1f8406eeb928ab0fe42c123ae40e94ec6c497dbf74e2c740226b9R40

Reply:
done